pkgs/tools/misc/calamares/default.nix

```diff
@@ -37,10 +37,14 @@ mkDerivation rec {
         -i com.github.calamares.calamares.policy
 
     sed -e 's,/usr/share/zoneinfo,${tzdata}/share/zoneinfo,' \
-        -i src/modules/locale/SetTimezoneJob.cpp
+        -i src/modules/locale/SetTimezoneJob.cpp \
+        -i src/modules/locale/Tests.cpp \
+        -i src/libcalamares/locale/TimeZone.cpp \
+        -i src/libcalamares/locale/zone-extractor.py
```

Comment:
And actually I think this is also just a code generator of [`src/libcalamares/locale/ZoneData_p.cxxtr`](https://github.com/calamares/calamares/blob/calamares/src/libcalamares/locale/ZoneData_p.cxxtr).